src/main/java/com/github/crimsondawn45/fabricshieldlib/initializers/FabricShieldLibClient.java

```diff
@@ -18,157 +14,153 @@
-         */
-        if(type.isAdvanced()) {
-
-            //Grab durability
-            if(stack.isDamaged()) {
-
-                for(int i = tooltip.size() - 1; i > 0; i--) {
-
-                    Text text = tooltip.get(i);
-                    String strText = text.getString();
-
-                    if(strText.startsWith("Durability")) {
-                        advanced.add(text);
-                        tooltip.remove(i);
-                        break;
-                    }
-                }
-            }
-
-            //Grab item id
-            for(int i = tooltip.size() - 1; i > 0; i--) {
-
-                Text text = tooltip.get(i);
-                String strText = text.getString().trim();
-
-                if(Identifier.isNamespaceValid(strText)) { //not sure if isnamespacevalid or ispathvalid
-                    advanced.add(text);
-                    tooltip.remove(i);
-                    break;
-                }
-            }
-
-            //Grab nbt string
-            if(!stack.getComponents().isEmpty()) {
-                for(int i = tooltip.size() - 1; i > 0; i--) {
-
-                    Text text = tooltip.get(i);
-                    String strText = text.getString();
-
-                    if(strText.startsWith("NBT: ")) {
-                        advanced.add(text);
-                        tooltip.remove(i);
-                        break;
-                    }
-                }
-            }
-        }
-
-        //Add disabled cooldown tooltip
-        tooltip.add(Text.literal(""));
-        tooltip.add(Text.translatable("fabricshieldlib.shield_tooltip.start").append(Text.literal(":")).formatted(Formatting.GRAY));
-
-        /*
-         * All of this is so if there is a .0 instead of there being a need for a
-         * decimal remove the .0
-         */
-        String cooldown = String.valueOf((Double)(cooldownTicks / 20.0));
-        char[] splitCooldown = cooldown.toCharArray();
-        if(splitCooldown.length >= 3) {
-
-            if(splitCooldown[2] == '0') {
-
-                if(!(splitCooldown.length >= 4)) {
-                    cooldown = String.valueOf(splitCooldown[0]);
-                }
-            }
-        }
-
-        tooltip.add(Text.literal(" " + cooldown)
-                        .formatted(Formatting.DARK_GREEN)
-                        .append(Text.translatable("fabricshieldlib.shield_tooltip.unit"))
-                        .append(Text.literal(" "))
-                        .append(Text.translatable("fabricshieldlib.shield_tooltip.end")));
-
-        //Append advanced info
-        if(type.isAdvanced()) {
-            tooltip.addAll(advanced);
-        }
-        return tooltip;
-    }
+	@SuppressWarnings("deprecation")
+	public static final SpriteIdentifier FABRIC_BANNER_SHIELD_BASE = new SpriteIdentifier(
+		SpriteAtlasTexture.BLOCK_ATLAS_TEXTURE,
+		Identifier.of(FabricShieldLib.MOD_ID, "entity/fabric_banner_shield_base")
+	);
+
```

Comment:
Could you potentially move these two to the test files?

Reply:
Sounds better to me if we put the test code into its own package

Reply:
Yep, sounds good to me